extensions/ql-vscode/src/databases/database-fetcher.ts

```diff
@@ -553,10 +571,18 @@ export async function convertGithubNwoToDatabaseUrl(
       }
     }
 
+    const databaseForLanguage = response.data.find(
+      (db: any) => db.language === language,
+    );
+    if (!databaseForLanguage) {
+      throw new Error(`No database found for language '${language}'`);
+    }
+
     return {
       databaseUrl: `https://api.github.com/repos/${owner}/${repo}/code-scanning/codeql/databases/${language}`,
       owner,
       name: repo,
+      commitOid: databaseForLanguage.commit_oid,
```

Comment:
It's worth noting that this is the commit oid at the point of checking the database info and it might have changed by the point that we download the database contents. However the download should happen immediately and I'm not sure we can do better anyway since there isn't an API endpoint that returns both the metadata and the contents.

Reply:
This is something we are aware of. The chance of this race condition happening is unlikely, but if it does, it's also not problematic. The only reason the `commit_oid` will be used is for checking updates to the database. If we run into this race condition, it's probably pretty likely that it's an active repository and we would have new commits coming in shortly after anyways.